app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java

```diff
@@ -472,12 +505,12 @@ private void removeProgressBar() {
     }
 
     private void removeProgressBarAndShowAlert() {
-        alert.setVisibility(View.VISIBLE); // Make alert "no finger" - visible
+        put_finger_alert.setVisibility(View.VISIBLE); // Make alert "no finger" - visible
```

Comment:
I think we want to add a check if the measurement started already (did the user put their finger at took it off after) or it is waiting to start (the user hasn't put their finger on yet)
We should raise different alerts in each case